Create 2025-04-03-content.md (#5244)
SumoLogic/sumologic-documentation: blog-cse/2025-04-03-content.md

---
title: April 3, 2025 - Content Release
image: https://help.sumologic.com/img/sumo-square.png
keywords:
  - log mappers
  - parsers
hide_table_of_contents: true    
---

import useBaseUrl from '@docusaurus/useBaseUrl';

> This Content Release Includes:
> - New and updated log mappers and parsers for Bitwarden, CommScope, Mimecast, and Sysdig Secure.
>     - Updates to Mimecast mappers are to support additional fields and events with new log parser.  

Reviewer: I think this is typically formatted as one sentence (@jpipkin1 is that correct?), so it'd be something like...

```suggestion
This content release includes new and updated log mappers and parsers for Bitwarden, CommScope, Mimecast (with support for additional fields and events with a new log parser), and Sysdig Secure.
```

Reviewer 2: It's okay as a bulleted list, @kimsauce. See [here](https://help.sumologic.com/release-notes-cse/2025/03/18/content/) for an example.

Reviewer: oh ok, thanks for clarifying!
